src/utils/app/__tests__/importExports.test.ts

```diff
@@ -222,6 +222,7 @@ const expectedConversation: Conversation = {
 describe('Export helpers functions', () => {
   it('Should return false for non-prompts data', () => {
     const testData = [{ id: 1 }];
+    //@ts-ignore
```

Comment:
it is not possible to fix it without ts ignore?

Reply:
Changed to type assertion ' as unknown as PromptsHistory' 
This is needed for the falsy test scenario.